### PR TITLE
v2v:add TC 'No useless info printed during conversion'

### PR DESCRIPTION
**Pull request by vwu-vera**
Please check bz #1693283

### CODE REVIEW

**Comment by vwu-vera** [comment]
(1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.no_useless_info.esx_67.rhev.rhv.it_default: STARTED
█ 100% [****************************************]]
 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.no_useless_info.esx_67.rhev.rhv.it_default: PASS (3536.65 s)

**Comment by vwu-vera** [comment]
@xiaodwan Please help to review. Thanks.:)

File: v2v/tests/cfg/function_test_esx.cfg [hunk 1]
```diff
@@ -714,6 +714,12 @@
             only esx_70
             version_requried = "[libvirt-7.0.0-1,)"
             main_vm = VM_NAME_ESX70_NONEXIST_ISO_V2V_EXAMPLE
+        - no_useless_info:
+            only esx_67
+            version_required = '[nbdkit-1.12.5-1,)'
+            main_vm = VM_NAME_RESUME_MAPPER_V2V_EXAMPLE
+            expect_msg = 'no'
+            msg_content = 'Exception AttributeError: "'module' object has no attribute 'dump_plugin'" in <module 'threading' from '/usr/lib64/python2.7/threading.pyc'> ignored'
```
Comment (xiaodwan) [inline v2v/tests/cfg/function_test_esx.cfg:722]: The msg should be a refined regular expression which can always match the msg in every v2v or python version.

In this case, msg_content = 'Exception AttributeError'  is enough. The other content is not important.

Comment (vwu-vera) [reply v2v/tests/cfg/function_test_esx.cfg]: Thanks for suggestion. Has been modified accordingly.

**Review by xiaodwan — APPROVED** [review]

**Pull request merged**